From a816b2943e6d2633fb101525c8ebc387f62bdf0f Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Tue, 30 Nov 2021 10:56:14 +0100
Subject: [PATCH 279/529] ARM: configs: multiv7_addons: enable simple drm
 framebuffer

Enable simple drm framebuffer.

Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Change-Id: Ib2dc267c423e4e3b8bad303fba05f81d2ad5f561
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/230492
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/280093
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -13,6 +13,7 @@ CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAUL
 CONFIG_DEBUG_PREEMPT=y
 CONFIG_DEBUG_SECTION_MISMATCH=y
 CONFIG_DNS_RESOLVER=y
+CONFIG_DRM_SIMPLEDRM=y
 CONFIG_DYNAMIC_DEBUG=y
 CONFIG_FORCE_MAX_ZONEORDER=12
 CONFIG_FUTEX=y
